jekyll/_cci2/server/v4.4/installation/phase-1-prerequisites.adoc

```diff
@@ -0,0 +1,889 @@
+
+CircleCI server v4.4 is installed as a Helm chart. The installation process is broken down into four phases. There is a validation step at the end of each phase, allowing you to confirm success before moving to the next phase. Depending on your requirements, phases 3 and 4 may include multiple steps. This installation guide assumes you have already read the xref:../overview/circleci-server-overview#[CircleCI server v4.4 overview].
+
+NOTE: In the following sections, replace any sections indicated by `< >` with your details.
+
+[#install-required-software]
+== 1. Install required software
+Download and install the following software before continuing:
+
+[.table.table-striped]
+[cols=4*, options="header", stripes=even]
+|===
+| Tool
+| Version
+| Used for
+| Notes
+
+| link:https://www.terraform.io/downloads.html[Terraform]
+| {terraformversion} or greater
+| Infrastructure Management
+|
+
+| link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[kubectl]
+| {kubectlversion} or greater
+| Kubernetes CLI
+|
+
+| link:https://helm.sh/[Helm]
+| {helmversion} or greater
+| Kubernetes Package Management
+|
+
+| link:https://github.com/databus23/helm-diff[Helm Diff]
+| {helmdiffversion} or greater
+| Helping with `values.yaml` changes and updates
+| Optional, but may help with troubleshooting between releases
+|===
+
+// Don't include this section in the GCP PDF.
+
+ifndef::env-gcp[]
+
+[#aws-required-software]
+=== AWS required software
+
+- link:https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html[AWS CLI]
+
+// Stop hiding from GCP PDF:
+
+endif::env-gcp[]
+
+// Don't include this section in the AWS PDF:
+
+ifndef::env-aws[]
+
+[#gcp-required-software]
+=== GCP required software
+
+- `gcloud` and `gsutil`. You can install and set-up these tools up by installing Google Cloud SDK. For further information refer to the link:https://cloud.google.com/sdk/docs/[Google Cloud SDK docs].
+
+endif::env-aws[]
+
+[#s3-compatible-storage-required-software]
+=== S3 compatible storage required software
+
+- Install and configure link:https://docs.min.io/docs/minio-client-quickstart-guide.html[MinIO CLI] for your storage provider.
+
+[#create-a-kubernetes-cluster]
+== 2. Create a Kubernetes cluster
+CircleCI server installs into an existing Kubernetes cluster. The application uses a large number of resources. Depending on your usage, your Kubernetes cluster should meet the following requirements:
+
+[.table.table-striped]
+[cols=5*, options="header", stripes=even]
+|===
+| Number of daily active CircleCI users
+| Minimum Nodes
+| Total CPU
+| Total RAM
+| NIC speed
+
+| < 500
+| 3
+| 12 cores
+| 32 GB
+| 1 Gbps
+
+| 500+
+| 3
+| 48 cores
+| 240 GB
+| 10 Gbps
+|===
```

Comment:
Looks like this will need to be updated for the new pre-reqs starting with 4.3: https://github.com/circleci/circleci-docs/pull/8545/files